stix_shifter_modules/reaqta/stix_transmission/connector.py

```diff
@@ -121,6 +142,21 @@ def modify_result(cls, data):
                     payload['eventName'] = event_name
                 
                 result['payload'] = cls.update_net_traffic_flow(payload, network_protocol)
+            
+            if result.get('payload') and result['payload'].get('data') and result['payload']['data'].get('tactics'):
+                tactic_list = result['payload']['data']['tactics']
+                technique = result['payload']['data']['technique']
+                result['payload']['data']['mod_tactics'] = {}
+                dict_list = []
+                for tactic in tactic_list:
+                    
+                    if (f"{tactic}" in TACTIC_NAME_MAPPING):
```

Comment:
`TACTIC_NAME_MAPPING.get(tactic)` more readable?

maybe you wanna change the dictionary name to simple lowercase

Reply:
you can save it in `stix_shifter_modules/reaqta/stix_translation/json` and read it from there. but that would be a overkill. this is already hard coded in the code. so no one will be able to change it anyway. you can probably add a debug log and add something in the comment

Reply:
I'll just modify it to use the lower case. There is already a comment describing it's purpose.

Reply:
`tactic_name_mapping.get(tactic)` ?

Reply:
Yes, that's what I've left it as.